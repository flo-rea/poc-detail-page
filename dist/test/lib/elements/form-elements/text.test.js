"use strict";
const React = require('react');
const renderer = require('react/lib/ReactTestRenderer');
const text_1 = require('../../../../src/lib/view/elements/form-elements/text');
jest.mock('react-dom');
describe('Text (label)', () => {
    const component = renderer.create(React.createElement(text_1.Text, {id: "item2", for: "item2"}));
    it('has correct default options', () => {
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
//# sourceMappingURL=text.test.js.map