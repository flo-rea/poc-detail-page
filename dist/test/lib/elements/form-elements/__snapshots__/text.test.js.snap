exports[`Text (label) has correct default options 1`] = `
<label
  for="item2"
  id="item2" />
`;
